extractors/python_functions/keyword_extraction/README.md

```diff
@@ -0,0 +1 @@
+This module extracts keywords from a text or passage, using unsupervised methods of keyword extraction. For this, it uses the `flashtext` package which takes input keywords that are needed to be matched and extracts the span of those keywords in the text.
```

Comment:
Please mention that this module will also working only with refinery > 1.6 because of the missing requirements.

Reply:
Added!